RecoEcal/EgammaCoreTools/interface/EcalClusterLazyTools.h

```diff
@@ -90,6 +90,13 @@ class EcalClusterLazyToolsBase {
   //  std::vector<int> flagsexcl_;
   //  std::vector<int> severitiesexcl_;
 
+ public:
+  inline const EcalRecHitCollection *getEcalEBRecHitCollection(void){return ebRecHits_;};
```

Comment:
these are more obviously prompting to suggest that the methods should be const-qualified.
Looking at other methods, there are actually quite a few more methods that should be made const-qualified as well.
This is something to put on the list of possible code cleanup one day.